README.md

```diff
@@ -314,7 +314,10 @@ module.exports = {
       {
         test: /\.lazy\.css$/i,
         use: [
-          { loader: 'style-loader', options: { injectType: 'lazyStyleTag' } },
+          {
+            loader: 'style-loader',
+            options: { injectType: 'lazySingletonStyleTag' },
```

Comment:
why not `lazyStyleTag` ?

Reply:
This section is about `lazySingletonStyleTag`.

Reply:
my bad  :)